lib/models/record.js

```diff
@@ -77,6 +78,33 @@ class Record extends BaseModel {
       (r) => new Server({ id: r.record_info.server_id })
     );
   }
+
+  async splits() {
+    if (this.rank > 1) return [];
+
+    let splits =
```

Comment:
I don't think you need the `this.access` stuff here at all - that basically tries to load from `fetchRecord` above which won't have the split info (and it isn't cacheing it how you might hope it would).